packages/sanity/src/core/store/_legacy/document/document-pair/operationArgs.ts

```diff
@@ -20,27 +20,40 @@ export const operationArgs = memoize(
       historyStore: HistoryStore
       schema: Schema
       serverActionsEnabled: Observable<boolean>
+      bundlePerspective?: string
```

Comment:
I noticed that this is not currently in use, are you planning to add support here? 
If not, i think it will be better to remove this to avoid confusions

Reply:
Good catch, thanks! I think I accidentally left this dangling from some initial exploration. The `idPair` parameter contains this information, so I don't think there's any reason to replicate it in the `ctx` parameter.